clusterloader2/testing/load/config.yaml

```diff
@@ -1,12 +1,33 @@
+#Constants
+{{$BIG_GROUP_SIZE := 250}}
+{{$MEDIUM_GROUP_SIZE := 30}}
+{{$SMALL_GROUP_SIZE := 5}}
+{{$PODS_PER_NODE := 30}}
+{{$NODES_PER_NAMESPACE := 100}}
+#Variables
+# bigRCs - 1/4 of all pods should be in big RCs.
+{{$bigRCs := MultiplyInt .Nodes 0.03}}
+# mediumRCs - 1/4 of all pods should be in medium RCs.
+{{$mediumRCs := MultiplyInt .Nodes 0.25}}
+# smallRCs - 1/2 of all pods should be in small RCs.
+{{$smallRCs := MultiplyInt .Nodes 3}}
+{{$namespaces := DivideInt (SubtractInt .Nodes 1) $NODES_PER_NAMESPACE | AddInt 1}}
+{{$bigRCsPerNamespace := DivideInt $bigRCs $namespaces}}
+{{$mediumRCsPerNamespace := DivideInt $mediumRCs $namespaces}}
+{{$smallRCsPerNamespace := DivideInt $smallRCs $namespaces}}
+{{$totalPods := MultiplyInt $PODS_PER_NODE .Nodes}}
+# saturationQPS - qps for RC operations should be an equivalent to 10 [pods/s].
+{{$saturationQPS := DivideFloat (AddInt $bigRCs $mediumRCs | AddInt $smallRCs) (DivideFloat $totalPods 20)}}
```

Comment:
Is it possible to change AddInt to take arbitrary no. of arguments? (to make this cleaner)

Reply:
At first I thought that template functions will required fixed number of parameters. However it turned out that template functions support variadic args.
Change implemented.